roles/client-microsoft-idp/files/ms2urn.php

```diff
@@ -0,0 +1,9 @@
+<?php
+$attributemap = array(
+    'sub' => 'urn:mace:dir:attribute-def:eduPersonPrincipalName',
+    'windowslive.id' => 'urn:mace:dir:attribute-def:uid',
+    'windowslive.displayName' => 'urn:mace:dir:attribute-def:cn',
+    'windowslive.givenName' => 'urn:mace:dir:attribute-def:givenName',
+    'windowslive.surname' => 'urn:mace:dir:attribute-def:sn',
+    'windowslive.userPrincipalName' => 'urn:mace:dir:attribute-def:mail',
```

Comment:
Bij de Google-IdP gebruiken we volgens mij <uid>@google-test.<env>.scz.lab.surf.nl 
Zullen we dat hier ook doen?

Reply:
Dat doen we hier ook, alleen op een andere plek